### DISCUSSION TITLE
YOLO badge bugged/ not received.

### DISCUSSION THREAD

**Discussion by Ayush-tring-to-code** [community/community#41164] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I have tried the possible ways to do it. Have also researched on the net.

But have not received the badge. Has GitHub changed the requirements for that? 

Can anyone guide me on that? I am not sure if there is a bug with my account or if am I doing something wrong.

PS: I am not hunting for it just wanted to learn....

**Comment by Priyanshu-Singh-1** [comment, marked as answer]
Ig there should not be any bug with the account.

Try the following options - 

Have you tried clicking on the Achievements link below your profile picture, maybe you have it and it's just hidden from view?
If that's the case, just unhide it.

Or

* Create a PR from your account.
* Assign a reviewer to the PR and this reviewer cannot be yourself.
* Merge the PR from your account without the approval of the reviewer.

**Comment by ghostinhershell** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Ayush-tring-to-code! The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by Kdmcbee88** [reply]
https://github.com/Kdmcbee88

**Reply by Kdmcbee88** [reply]
> Hey there!  👋 
> 
> Thanks for posting in the GitHub Community, @Ayush-tring-to-code! The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
> 
> I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by Kushal997-das** [comment]
Hello I got the answer and it's perfectly working -

STEP 1 : Choose a normal repo. of yours remember in that repo there shouldn't be any `.github` folder or any auto running things not present .

STEP 2 : Now go to that repository settings add another account as collaborator . (Settings->collaborator->Add people).

STEP 3 : You have to Create a Pull request (PR) on this repository.

STEP 4 : Add that collaborator account as reviewer.

STEP 5 : Then You have to merge your opned PR from Your account only not collaborator , Without reviewing the PR .

I hope it gonna help you guys , Thanks :)
